scripts/install-windows-prereqs.ps1

```diff
@@ -14,16 +14,16 @@ Param(
     [string]$VSBuildToolsHash = '',
     [string]$Clang7URL = 'http://releases.llvm.org/7.0.1/LLVM-7.0.1-win64.exe',
     [string]$Clang7Hash = '672E4C420D6543A8A9F8EC5F1E5F283D88AC2155EF4C57232A399160A02BFF57',
-    [string]$IntelPSWURL = 'http://registrationcenter-download.intel.com/akdlm/irc_nas/16607/Intel%20SGX%20PSW%20for%20Windows%20v2.7.101.2.exe',
-    [string]$IntelPSWHash = 'AF669A4593411E9AABCE18838C91003866DDDEDAC5BEEC61DE160025008B0A19',
+    [string]$IntelPSWURL = 'http://registrationcenter-download.intel.com/akdlm/irc_nas/16766/Intel%20SGX%20PSW%20for%20Windows%20v2.8.100.2.exe',
```

Comment:
Instead of continuing to use the Intel registration center links here, can we address #3015 as well by using the Microsoft Update links? We might want to reach out to Francisco about this since I don't know whether the latest packages have already been ingested to WU.

Reply:
Follow-up: I did a search of available WU packages and didn't see the updated package versions, feel free to ignore this suggestion unless you've managed to get the links from Francisco et. al.

Reply:
Thanks for checking that out @CodeMonkeyLeet :)